ghost/external-media-inliner/package.json

```diff
@@ -22,7 +22,7 @@
     "mocha": "10.2.0",
     "sinon": "15.2.0"
   },
-"dependencies": {
-    "mime-types": "2.1.35"
-}
+  "dependencies": {
+    "file-type": "16"
```

Comment:
should be 16.0.0 ?

Reply:
`16.5.4` actually :D `yarn add -E file-type@16` was the culprit.